src/title/titleSelectors.js

```diff
@@ -4,12 +4,11 @@ import { isStreamOrTopicNarrow, streamNameOfNarrow } from '../utils/narrow';
 import { getSubscriptionsByName } from '../subscriptions/subscriptionSelectors';
 
 /**
- * Background color to use for the app bar in narrow `narrow`.
+ * The stream's color for the given stream or topic narrow.
  *
- * If `narrow` is a stream or topic narrow, this is based on the stream color.
- * Otherwise, it takes a default value.
+ * Gives undefined for narrows that are not stream or topic narrows.
  */
-export const getTitleBackgroundColor = (state: GlobalState, narrow: Narrow) => {
+export const getStreamColorForNarrow = (state: GlobalState, narrow: Narrow) => {
```

Comment:
With its new clearer name, I think this also is best moved to a different file: `subscriptionSelectors.js`, where it will fit right in.

That also lets us remove `titleSelectors.js`, which never made as much sense as the selectors files like `subscriptionSelectors.js` that are clearly about a particular part of the app's data model.